src/combined_ice_ocean_driver.F90

```diff
@@ -41,13 +43,16 @@ module combined_ice_ocean_driver
   logical :: intersperse_ice_ocn !< If true, intersperse the ice and ocean thermodynamic and
                               !! dynamic updates.  This requires the update ocean (MOM6) interfaces
                               !! used with single_MOM_call=.false. The default is false.
+  logical :: use_intersperse_bug !< If true, use a bug in the intersperse option where the ocean 
```

Comment:
I am not completely sure of this, but it looks to me like the bug was the omission of these calls, so when USE_INTERSPERSE_BUG is true, the bug is being fixed.  We should change the logic around so that the use_bug = .true. option is the one with the bug and the one where this is false fixes the bug, to avoid user confusion.

Reply:
Changed to be the correct logic.